src/plugins/syscalls/win.h

```diff
@@ -3453,7 +3467,10 @@ SYSCALL(NtUserCallMsgFilter, NTSTATUS, 0);
 SYSCALL(NtUserCallNextHookEx, NTSTATUS, 0);
 SYSCALL(NtUserCallNoParam, NTSTATUS, 0);
 SYSCALL(NtUserCallOneParam, NTSTATUS, 0);
-SYSCALL(NtUserCallTwoParam, NTSTATUS, 0);
+SYSCALL(NtUserCallTwoParam, NTSTATUS, 3,
```

Comment:
Please move this above to be placed with the other defined syscalls

Reply:
Done